### ISSUE TITLE
Installing composer packages

### ISSUE THREAD

**Issue by jasonujmaalvis** [paliarush/vagrant-magento#95]
Hello,

How do I install a composer package? When I navigate to vagrant-magento/magento2ce and enter the following command I get missing PHP extensions error:

```
cd /vagrant-magento/magento2ce
composer require jasonalvis/magento2-core

Problem 1
    - The requested PHP extension ext-mcrypt * is missing from your system. Install or enable     PHP's mcrypt extension.

  Problem 2
    - The requested PHP extension ext-intl * is missing from your system. Install or enable PHP's intl extension.
```

I'm not sure if I'm in the correct directory. I've also tried it in the root directory /vagrant-magento but it just creates a vendors folder outside of the magento folder.

Thank you,
Jason

**Comment by paliarush** [comment]
If you need to install Magento from composer packages, please use this branch https://github.com/paliarush/magento2-vagrant-for-developers/tree/Composer-Create-Project  , it already includes functionality that you need (will be merged to main branch soon).

Make sure to have following section in `etc/config.yaml` of vagrant project:

```
# Possible values: git, composer [To apply changes: init_project.sh -fc]
checkout_source_from: "git"

composer_project:
  # [To apply changes: init_project.sh -fc]
  name: "magento/project-community-edition"
  # Make sure to specify valid keys for this repository in etc/composer/auth.json
  url: "https://repo.magento.com"
```

It is possible to specify project name with specific version. Then just run `bash ./init_project.sh -fc`

If the above is not what you need, to overcome error that you mentioned, just add `--ignore-platform-reqs` option while running `composer` commands.

**Comment by paliarush** [comment]
Merged to main branch (2.0) in scope of "Composer create project #96"

**Comment by jasonujmaalvis** [comment]
Thank you for your response! I ended up using the `--ignore-platform-reqs` option which worked perfectly.
